Cargo.toml

```diff
@@ -221,3 +221,4 @@ zksync_crypto_primitives = { path = "core/lib/crypto_primitives" }
 zksync_node_framework = { path = "core/node/node_framework" }
 zksync_eth_watch = { path = "core/node/eth_watch" }
 zksync_shared_metrics = { path = "core/node/shared_metrics" }
+zksync_block_reverter = { path = "core/node/block_reverter"}
```

Comment:
Same as #1636, you should include this crate in `workspace.members`.

Reply:
Could u pls explain why we need to include this into `workspace.members`?